src/test/java/uk/gov/justice/digital/converter/dms/DMS_3_4_6Test.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.converter.dms;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DMS_3_4_6Test {
+
+    private static SparkSession sparkSession;
+
+    @BeforeAll
+    public static void beforeAll(){
+        sparkSession = SparkSession.builder().master("local[*]").getOrCreate();
```

Comment:
I think this is fine for now. In the long run we can factor this out and have it started across the test suite instead and perhaps then look at moving these out into a separate set of tests e.g. `it` if they prove to be slow (athough starting / stopping spark once across the entire suite should help).